NOnion/Directory/TorDirectory.fs

```diff
@@ -217,11 +168,34 @@ type TorDirectory =
                     "/tor/status-vote/current/consensus"
                     false
 
+            let descriptorsStream = TorStream(circuit)
+            do! descriptorsStream.ConnectToDirectory() |> Async.Ignore
+
+            let descriptorsHttpClient =
+                TorHttpClient(descriptorsStream, "127.0.0.1")
```

Comment:
lets put this in a constant, as you have used it more than once

Reply:
Fixed